Danki2/Assets/Scripts/Actor/Actor.cs

```diff
@@ -4,15 +4,30 @@
 
 public class Actor : MonoBehaviour
 {
+    [HideInInspector]
+    public Stats baseStats = new Stats(0);
```

Comment:
Make this a readonly property (don't know whether Java syntax is the same):
```c#
public Stats baseStats { get; } = new Stats(0);
```

Reply:
Unfortunately this messes with the serialisation, unity can't serialise properties

Reply:
Ah yeah fair enough - not the end of the world anyway